tests/x11/gnote/gnote_rename_title.pm

```diff
@@ -26,8 +26,8 @@ sub run {
     send_key "up";
     type_string "new title-opensuse\n";
     send_key "ctrl-tab";    #jump to toolbar
-    send_key "ctrl-tab" if is_sle('15+');    #jump to toolbar for SLED15
-    send_key "ret";                          #back to all notes interface
+    send_key "ctrl-tab" if is_sle('15+') && get_var("WAYLAND");    #jump to toolbar for SLED15
```

Comment:
Why does Wayland cause an additional key press to be required?

Reply:
I just noticed the difference between Wayland and X11 and don't know why.